src/catch2/internal/catch_clara.hpp

```diff
@@ -53,7 +54,27 @@ namespace Catch {
             ShortCircuitSame
         };
 
+        struct accept_many_t {};
+        constexpr accept_many_t accept_many {};
+
         namespace Detail {
+            struct fake_arg {
+                template <typename T>
+                operator T();
+            };
+
+            template<typename... Ts> struct make_void { using type = void; };
```

Comment:
I am 99% sure that we already have this somewhere else, including a `_t` suffixed type alias. Since Clara is no longer a separate project, it should reuse the one that Catch2 already has.